apps/framework-cli/src/cli/routines.rs

```diff
@@ -260,6 +261,16 @@ pub async fn start_development_mode(project: Arc<Project>) -> anyhow::Result<()>
     let (framework_object_versions, version_syncs) =
         initialize_project_state(project.clone(), &mut route_table).await?;
 
+    {
+        let mut client = get_pool(&project.clickhouse_config).get_handle().await?;
```

Comment:
Is the client thread safe?

feels a weird location to have that in a closure

Reply:
I think the pool is thread safe but the client is not.